### PR TITLE
Added password field to user class and its constructor, fixed all tes…

### PR DESCRIPTION
**Pull request by Faramawy009**
…ts broken by adding a hardcoded "password" into their constructor calls

### CODE REVIEW

**Review by jeoratz — APPROVED** [review]

**Review by cvu0014 — APPROVED** [review]

File: src/main/java/codeu/controller/LoginServlet.java [hunk 1]
```diff
@@ -74,7 +74,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      User user = new User(UUID.randomUUID(), username, Instant.now());
+      // TODO: add password
+      User user = new User(UUID.randomUUID(), username, null, Instant.now());
+//      User user = new User(UUID.randomUUID(), username, Instant.now());
```
Comment (gauravmishra) [inline src/main/java/codeu/controller/LoginServlet.java:79]: readability nit: indentation - the // should start just below the start of the previous line. Ditto below...

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 2]
```diff
@@ -65,7 +65,9 @@ public PersistentDataStore() {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         String userName = (String) entity.getProperty("username");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        User user = new User(uuid, userName, creationTime);
+        // TODO: get password from Datastore
+        User user = new User(uuid, userName, null, creationTime);
+//        User user = new User(uuid, userName, creationTime);
```
Comment (gauravmishra) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java:70]: Ditto (indentation)

File: src/main/java/codeu/controller/LoginServlet.java (hunk 1 above)
Comment (gauravmishra) [reply src/main/java/codeu/controller/LoginServlet.java:79]: Better: you can just remove this line.

**Review by gauravmishra — APPROVED** [review]
Left a couple of readability nits. Overall, great job on the implementation! :)

**Pull request merged**